torch/quantization/custom_module_class_mappings.py

```diff
@@ -0,0 +1,55 @@
+OBSERVED_CUSTOM_MODULE_CLASS_MAPPINGS = dict()
```

Comment:
Are these mappings global?

Reply:
yes, and we have global getter and setter functions listed below